docs/concepts.rst

```diff
@@ -5,9 +5,9 @@ The Product Delivery Model
 -----------------------------
 CumulusCI extends the concepts of the `Org Development Model <https://trailhead.salesforce.com/en/content/learn/modules/org-development-model>`_ and the `Package Development Model <https://trailhead.salesforce.com/en/content/learn/modules/sfdx_dev_model>`_ to form a new model, which we call the Product Delivery Model.
```

Comment:
"CCI is good for moving parts at scale. It allows for multiple tiers/branches of dev and prod testing."